.circleci/config.yml

```diff
@@ -16,11 +16,9 @@ common: &common
     - run:
         name: upload coverage report
         command: |
-          if [[ "$UPLOAD_COVERAGE" != 0 ]]; then
-              PATH=$HOME/.local/bin:$PATH
-              pip install --user codecov
-              coverage xml
-              ~/.local/bin/codecov --required -X search gcov pycov -f coverage.xml --flags $CIRCLE_JOB
+          if [[ "${TOXENV%-coverage}" != "$TOXENV" ]]; then
+            .tox/$TOXENV/bin/coverage xml
+            bash <(curl -s https://codecov.io/bash) -Z -X gcov -X coveragepy -X fix -X search -X xcode -f coverage.xml -F $CIRCLE_JOB
```

Comment:
Worked in https://circleci.com/gh/lock8/pinax-stripe/1272 (using codecov-python).
Apparently a problem with codecov-bash?!